### PR TITLE
REP-5033 Updating the mock Keystone service

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
-  "author": "Open Repose Team <ReposeCore@Rackspace.com> (http://www.openrepose.org/)",
+  "name": "service-mocks",
+  "version": "1.0.0",
+  "description": "Simple services that mock the behavior of more complex services like OpenStack Identity.",
+  "author": "Damien Johnson <damien.johnson@rackspace.com>",
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: I'm not sure we want to change the name, major version, description, and author on this version. Also the readme got removed.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+    "body-parser": "^1.16.0",
+    "date-utils": "^1.2.21",
+    "express": "^4.14.1",
+    "libxmljs": "^0.18.4",
+    "uuid": "^3.0.1"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/rackerlabs/repose"
+    "url": "https://github.com/rackerlabs/repose-perf-infrastructure"
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: The repository url probably shouldn't change either.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 3]
```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
+  "name": "service-mocks",
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: The name should probably stay too.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+  "version": "1.0.0",
+  "description": "Simple services that mock the behavior of more complex services like OpenStack Identity.",
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: This seems to be a little more generic than we want right now. At least until we move all of them into a single build.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 5]
```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
+  "name": "service-mocks",
+  "version": "1.0.0",
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: Should we bump this to a full major version release or are we only at a minor?
This feels more like a minor.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json [hunk 6]
```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: Why remove the readme?

Comment (dmnjohns) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: I'm going to answer all of these at once.

I just copypasta'd the package.json file.

I'll fix it up.

File: repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json (hunk 5 above)
Comment (dmnjohns) [inline repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json]: Honestly, I don't think we'll be publishing this, so it really doesn't make much difference.

I like to start initial releases at a major, which is how I treated this module.

I'll drop it to `0.2.0` if you want, though.

**Comment by wdschei** [comment]
Just waiting on the tests to run... that don't actually use these changes... so it better not break anything.

**Pull request merged**